### PR TITLE
Update CI files, add dependent repositories

### PR DESCRIPTION
**Pull request by Bolodya1997**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -121,16 +93,18 @@ jobs:
         with:
           go-version: 1.15
       - name: Install proto-gen-go
-        run: go get -u github.com/golang/protobuf/protoc-gen-go@v1.4.2
-      - name: Generate protobufs
+        run: go get -u github.com/golang/protobuf/protoc-gen-go@v1.3.3
```
Comment (denis-tingajkin) [inline .github/workflows/ci.yaml:96]: Please use go get -u github.com/golang/protobuf/protoc-gen-go@v1.4.2

Comment (denis-tingajkin) [inline .github/workflows/ci.yaml:96]: Please use go get -u github.com/golang/protobuf/protoc-gen-go@v1.4.2

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -121,16 +93,18 @@ jobs:
...
+      - name: Install proto-gen-go
+        run: go get github.com/searKing/golang/tools/cmd/go-syncmap
```
Comment (denis-tingajkin) [inline .github/workflows/ci.yaml:98]: We do not need this for api.

File: .github/workflows/update-dependent-repositories-gomod.yaml [hunk 3]
```diff
@@ -7,9 +7,22 @@ on:
...
+          - sdk-vppagent
+          - sdk-kernel
+          - sdk-sriov
+          - cmd-nsmgr
+          - cmd-nsmgr-proxy
+          - cmd-registry-memory
+          - cmd-registry-proxy-dns
+          - cmd-forwarder-vppagent
+          - cmd-forwarder-sriov
+          - cmd-nsc
```
Comment (denis-tingajkin) [inline .github/workflows/update-dependent-repositories-gomod.yaml:25]: This is wrong for api. Please use only sdk

**Review by denis-tingajkin — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Bolodya1997) [reply .github/workflows/ci.yaml]: Fixed

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (Bolodya1997) [reply .github/workflows/ci.yaml]: Fixed

File: .github/workflows/update-dependent-repositories-gomod.yaml (hunk 3 above)
Comment (Bolodya1997) [reply .github/workflows/update-dependent-repositories-gomod.yaml]: Fixed

**Pull request merged**